cs.html

```diff
@@ -961,689 +1090,685 @@ <h2>Constructors</h2>
-                 <a>[[\QuicTransportWritableStreams]]</a> internal slot.</li>
-               </ol>
-               <div class="note">The actual transmission of data occurs in
-               parallel. If sending data leads to a QUIC-level error, the
-               application will be notified asynchronously through the
-               <code><a>QuicTransportBase</a></code>'s <code><a>onerror</a></code>
-               EventHandler.</div>
-              </ol>
-              <table class="parameters">
-                <tbody>
-                  <tr>
-                    <th>Parameter</th>
-                    <th>Type</th>
-                    <th>Nullable</th>
-                    <th>Optional</th>
-                    <th>Description</th>
+                    <td class="prmDesc">The host to connect to.</td>
                   </tr>
                   <tr>
-                    <td class="prmName">data</td>
-                    <td class="prmType"><code>QuicStreamWriteParameters</code></td>
+                    <td class="prmName">port</td>
+                    <td class="prmType"><code><a>unsigned short</a></code></td>
                     <td class="prmNullFalse"><span role="img" aria-label=
                     "False">&#10008;</span></td>
                     <td class="prmOptFalse"><span role="img" aria-label=
                     "False">&#10008;</span></td>
-                    <td class="prmDesc"></td>
+                    <td class="prmDesc">The port to connect to.</td>
                   </tr>
                 </tbody>
               </table>
-              <div>
-                <em>Return type:</em> <code>void</code>
-              </div>
             </dd>
-            <dt><dfn><code>abortWriting</code></dfn></dt>
-            <dd>
-              <p>A hard shutdown of the <code><a>QuicWritableStream</a></code>. It may be called
-              regardless of whether the <code><a>QuicWritableStream</a></code>
-              was created by the local or remote peer. When the <code>abortWriting()</code>
-              method is called, the user agent <em class="rfc2119" title="MUST">MUST</em>
-              run the following steps:</p>
+          </dl>
+        </section>
+      </div>
+    </section>
+  </section>
+
+  <section id="outgoing-stream*">
+    <h2>Interface Mixin <dfn>OutgoingStream</dfn></h2>
+    <p>
+      An OutgoingStream is a stream that can be written to,
+      as either a <code>SendStream </code>or a <code>BidirectionalStream</code>
+    </p>
+    <div>
+      <pre class="idl">
+      [ Exposed=Window ]
+interface mixin OutgoingStream {
+    readonly attribute boolean writable;
+    readonly attribute unsigned long writeBufferedAmount;
+    readonly attribute Promise&lt;StreamAbortInfo&gt; writingAborted;
+    void write (StreamWriteParameters data);
+    void abortWriting (StreamAbortInfo abortInfo);
+    Promise&lt;void&gt; waitForWriteBufferedAmountBelow(unsigned long threshold);
+};</pre>
+      <section>
```

Comment:
For some reason "Interface Mixin OutgoingStream" & the "Interface Mixin IncomingStream" aren't split up into subsections appropriately like in Transport Mixins for example:

https://rawgit.com/w3c/webrtc-quic/web-transport/cs.html#outgoing-stream*

I'm assuming it's an html issue, missing a closing element or something related.